staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -117,10 +117,22 @@ func MergeAPIResourceConfigs(
 			return nil, err
 		}
 		if enabled {
+			// enable the groupVersion for "group/version=true" and "group/version/resource=true"
 			resourceConfig.EnableVersions(groupVersion)
-		} else {
+		} else if len(tokens) == 2 {
+			// disable the groupVersion only for "group/version=false", not "group/version/resource=false"
 			resourceConfig.DisableVersions(groupVersion)
 		}
+
+		if len(tokens) < 3 {
+			continue
+		}
```

Comment:
Disabling individual resources is a bad idea.  I accept the need for extensions, but I don't want to accept a three token resource for anything outside of the extensions group and perhaps even just specific resources in here.  Please reject any groupversion other than extensions/v1beta1 as invalid.

Reply:
I know this is a generic library.  I would have this hardcoding as preferable to opening it up.

Reply:
> Please reject any groupversion other than extensions/v1beta1 as invalid.



> I know this is a generic library. I would have this hardcoding as preferable to opening it up.

Doing this will prevent anyone from extending the bad pattern and will make it possible to drop the more detailed enablement requests (which we should have removed before and that would make this current pull impossible to have writtent).

Reply:
limited to extensions/v1beta1